lldb/include/lldb/API/SBProgress.h

```diff
@@ -51,6 +51,10 @@ class LLDB_API SBProgress {
   SBProgress(const char *title, const char *details, uint64_t total_units,
              SBDebugger &debugger);
 
+#ifndef SWIG
```

Comment:
Why the move operator for only Swig? Wouldn't this be useful if say DAP wanted to grab it's own progress object and then potentially move it?

Reply:
This is a header guard that prevents SWIG from processing the move constructor. SWIG generates the bindings by reading the API headers and doesn't handle move constructors well. SBStream does the same thing.

When the headers are installed for distribution, this header guard is removed with `unifdef`.